src/сomponents/Footer.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { Filter } from '../App';
+
+type Props = {
+  todos: Todo[];
+  filter: Filter;
+  setFilter: React.Dispatch<React.SetStateAction<Filter>>;
+  handleClearCompleted: () => void;
+  loading: boolean;
+};
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  filter,
+  setFilter,
+  handleClearCompleted,
+  loading,
+}) => {
+  const activeTodos = todos.filter(todo => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodos} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={`filter__link ${filter === Filter.ALL ? 'selected' : ''}`}
+          data-cy="FilterLinkAll"
+          onClick={() => setFilter(Filter.ALL)}
+        >
+          All
+        </a>
+        <a
+          href="#/active"
+          className={`filter__link ${filter === Filter.ACT ? 'selected' : ''}`}
+          data-cy="FilterLinkActive"
+          onClick={() => setFilter(Filter.ACT)}
+        >
+          Active
+        </a>
+        <a
+          href="#/completed"
+          className={`filter__link ${filter === Filter.COMP ? 'selected' : ''}`}
+          data-cy="FilterLinkCompleted"
+          onClick={() => setFilter(Filter.COMP)}
+        >
+          Completed
+        </a>
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method